### PR TITLE
Feature/779 fix styles not in less

### LINKED ISSUE #779: Fix styling written directly to css file
Some css styling (written by me, earlier) were written directly to `app.css` instead to their respective `.less` file. Fix this.

### PR DESCRIPTION
**Pull request by yotammanor**
#### Relating issues 
Issue link and do the pull request resolves it. e.g:
- Resolves #779 

#### Change list
- Moved some styling from css file to less file.
- fixed a minor styling bug.

### CODE REVIEW

File: less/kikar.less [hunk 1]
```diff
@@ -25,4 +25,72 @@
...
+
+.img-fb-status-attachment {
+    height: auto;
+    margin: auto;
+    border: solid;
+    border-width: thin;
+    border-color: #dddddd;
+}
+
+@media (min-width: 768px) {
```
Comment (alonisser) [inline less/kikar.less]: This is missing the point of using less.. should be cascaded in the original class above, same for the next media query

File: less/kikar.less [hunk 2]
```diff
@@ -25,4 +25,72 @@
...
+
+.embed-container {
+    position: relative;
+    padding-bottom: 56.25%;
+    height: 0;
+    overflow: hidden;
+    max-width: 100%;
+}
+
+.embed-container iframe, .embed-container object, .embed-container embed {
```
Comment (alonisser) [inline less/kikar.less]: missing the point of using less should be part of the class above someClass:{

variousettings:11px;
iframe, object, embed {
anothe setting:12px
}
}

File: less/kikar.less (hunk 1 above)
Comment (alonisser) [inline less/kikar.less]: Also just verifying that any new less code was also compiled to ss

Comment (yotammanor) [inline less/kikar.less]: I am not sure I understand the second comment. I'm compiling all changes made in `less` files.

**Comment by yotammanor** [comment]
@alonisser are the changes approved? :)

File: static/css/app.css [hunk 3]
```diff
@@ -8198,12 +8196,12 @@ body.infinity-protocol #content-main > div > div.row > div.span3 {
   border-color: #dddddd;
 }
 @media (min-width: 768px) {
```
Comment (alonisser) [inline static/css/app.css]: Is this auto generated from less file? because classes seem to be doubled

File: static/css/app.css [hunk 4]
```diff
@@ -8198,12 +8196,12 @@ body.infinity-protocol #content-main > div > div.row > div.span3 {
...
-  .img-fb-status-attachment {
+  .img-fb-status-attachment .img-fb-status-attachment {
```
Comment (alonisser) [inline static/css/app.css]: Is this auto generated from less file? because classes seem to be doubled

File: less/kikar.less [hunk 5]
```diff
@@ -22,75 +22,73 @@
...
-}
+  height: auto;
+  margin: auto;
+  border: solid;
+  border-width: thin;
+  border-color: #dddddd;
 
-@media (min-width: 768px) {
+  @media (min-width: 768px) {
     .img-fb-status-attachment {
```
Comment (alonisser) [inline less/kikar.less]: You don't need the double class inside, you already get it from the outside.. maybe that's why the classes are doubled?

File: less/kikar.less [hunk 6]
```diff
@@ -22,75 +22,73 @@
...
-        max-width: 50%;
+      max-width: 50%;
     }
-}
-
-@media (max-width: 767px) {
+  }
+  @media (max-width: 767px) {
     .img-fb-status-attachment {
```
Comment (alonisser) [inline less/kikar.less]: Same as above

File: less/kikar.less (hunk 5 above)
Comment (yotammanor) [inline less/kikar.less]: That was it. Pushing a fix in a minute.

**Comment by alonisser** [comment]
@yotammanor You saw the other changes/questions?

**Comment by yotammanor** [comment]
@alonisser I fixed the classes that got doubled, if that's what you mean.

**Pull request merged**